src/sentry/tagstore/multi/backend.py

```diff
@@ -0,0 +1,175 @@
+"""
+sentry.tagstore.multi.backend
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:copyright: (c) 2010-2017 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
+"""
+
+from __future__ import absolute_import
+
+import six
+
+import random
+
+from sentry.tagstore.base import TagStorage
+from sentry.utils.imports import import_string
+
+
+class MultiTagStorage(TagStorage):
+    def __init__(self, backends, read_selector=random.choice, **kwargs):
+        assert backends, "you should provide at least one backend"
+
+        self.backends = []
+        for backend, backend_options in backends:
+            if isinstance(backend, six.string_types):
+                backend = import_string(backend)
+            self.backends.append(backend(**backend_options))
+
+        self.read_selector = read_selector
+
+        super(TagStorage, self).__init__(**kwargs)
+
+    def _call_all_backends(self, func, *args, **kwargs):
```

Comment:
Would love if this happened in a thread, since we're ignoring the output anyhow, we should make it a fire and forget.

Reply:
Ya that was my plan, I only took it off WIP so I could get Travis going here.